tce/tools/scripts/install_llvm_10.sh

```diff
@@ -82,6 +82,7 @@ cmake ../llvm/ -DLLVM_ENABLE_PROJECTS=clang \
     -G "Unix Makefiles" \
     $LLVM_BUILD_MODE\
     -DCMAKE_INSTALL_PREFIX=$TARGET_DIR \
+    -DLLVM_TARGETS_TO_BUILD="X86" \
```

Comment:
What about TCE(LE) targets?

Reply:
Endianness only affects memory based operations (LD/ST). Since the instruction-selector of TCE can already take care of this (thanks to re-targettable instruction set), we do not need any special handling.

The testsuite seem to pass for both LE and BE architectures.

If I am missing something here, please correct me.

Reply:
I thought you should also enable TCE, TCELE and now TCELE64 targets In cmake?

Reply:
The TCE backend is built separately and loaded dynamically. So, TCE backend is not registered to the LLVM at all in this stage.

Reply:
Yes. But there is a stub at Clang side, I suppose it is always built in then.